flake.nix

```diff
@@ -0,0 +1,45 @@
+{
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+    rust-overlay.url = "github:oxalica/rust-overlay";
+  };
```

Comment:
We can add [nix-systems](https://github.com/nix-systems/default-linux) as inputs to make use of it for other architectures as well. For a starting point, lets take look at the hyprland flake.nix. [hyprwm/hyprland/flake.nix](https://github.com/hyprwm/Hyprland/blob/main/flake.nix).

Reply:
added